docs/reference/argocd.md

```diff
@@ -847,6 +848,12 @@ LogFormat | text | The log format to be used by the ArgoCD Repo Server. Valid op
 ExecTimeout | 180 | Execution timeout in seconds for rendering tools (e.g. Helm, Kustomize)
 Env | [Empty] | Environment to set for the repository server workloads
 Replicas | [Empty] | The number of replicas for the ArgoCD Repo Server. Must be greater than or equal to 0.
+Volumes | [Empty] | Add addition volumes to the repo server deployment.
+VolumeMounts | [Empty] | Add addition volume mounts to the repo server deployment.
+InitContainers | [Empty] | List of initialization containers for the repo server deployment.
```

Comment:
`List of initContainers` . This is optional as well right ? Can we have a column to tell if its optional or mandatory in addition to the default value ?

Reply:
Yes. I added `This field is optional.` at the end. Adding a new column would be a significant effort since we need to update it for all component tables. We can address this in a separate PR.

Reply:
Sounds good. Thanks.